backend/app/graphql/meal_request.py

```diff
@@ -17,20 +23,20 @@ class MealRequestTypeInput(graphene.InputObjectType):
 
 class MealTypeInput(graphene.InputObjectType):
     portions = graphene.Int(required=True)
-    dietary_restrictions = graphene.String(required=True)
-    meal_suggestions = graphene.String(required=True)
+    dietary_restrictions = graphene.String(default_value=None)
+    meal_suggestions = graphene.String(default_value=None)
```

Comment:
I changed these from required to optional, since it seems possible that a requestor would make a request where specific dietary restrictions are not applicable, and/or they have no suggestions for possible meals.